lib/matplotlib/lines.py

```diff
@@ -568,7 +568,7 @@ def get_window_extent(self, renderer):
                                  ignore=True)
         # correct for marker size, if any
         if self._marker:
-            ms = (self._markersize / 72.0 * self.figure.dpi) * 0.5
```

Comment:
Yes.  The assumption that markers only fill (-0.5, 0.5) is only true for circles.  Ticks fill (0, 1) or (-1, 0) depending on "in" or "out".